resources/logs/check_results.sh

```diff
@@ -66,58 +66,92 @@ check_results_02 () {
 }
 
 check_results_03a () {
-    if [ -f "${section_03_dir}/methylation_summary.RData" ]; then
-        echo "Methylation_summary.RData is present"
-    else
-        echo "Problem: methylation_summary.RData is absent"
-        exit 1
-    fi
+
+	if [ -f "${section_03_dir}/methylation_summary.RData" ]; then
+		echo "Methylation_summary.RData is present"
+	else
+		echo "Problem: methylation_summary.RData is absent"
+		exit 1
+	fi
 
     if [ -f "${section_03_dir}/cohort_descriptives_commonids.RData" ]; then
-        echo "cohort_descriptives_commonids.RData is present"
-    else
-        echo "Problem: cohort_descriptives_commonids.RData is absent"
-        exit 1
-    fi
+		echo "cohort_descriptives_commonids.RData is present"
+	else
+		echo "Problem: cohort_descriptives_commonids.RData is absent"
+		exit 1
+	fi
 
+	if [ "${cellcounts_required}" = "yes" ]; then
+		if [ -f "${section_03_dir}/cellcounts_summary.txt" ]; then
+			echo "Summary statistics of cell counts are present"
+		else
+			echo "Problem: summary statistics of cell counts are absent"
+			exit 1
+		fi
 
-    if [ "${cellcounts_required}" = "yes" ]; then
-        if [ -f "${section_03_dir}/cor_plot.pdf" ]; then
-          echo "Correlation plot of observed vs predicted cell counts is present"
-        else
-          echo "Problem: correlation plot of observed vs predicted cell counts is absent"
-          exit 1
-        fi
-    fi
+		if [ -f "${section_03_dir}/cellcounts_plot.pdf" ]; then
+			echo "Plots of cell counts are present"
+		else
+			echo "Problem: plots of cell counts are absent"
+			exit 1
+		fi
+	else
+		echo "Message: since cellcounts_required is not required, no output for cellcounts_summary.txt and cellcounts_plot.pdf for cell counts."
+	fi
 
-    if [ "${cellcounts_required}" = "yes" ]; then
```

Comment:
Aren't we forcing all cohorts to recalulcate the cell composition variables? So the check for the plot should just be based on whether the cohort has observed counts to plot against? i.e. if measured_cellcounts != "NULL"